.ci/validation.sh

```diff
@@ -1260,26 +1260,27 @@ openrewrite-recipes)
   PROJECT_ROOT="$(pwd)"
   export MAVEN_OPTS="-Xmx4g -Xms2g"
 
-  cd /tmp
-  git clone https://github.com/checkstyle/checkstyle-openrewrite-recipes.git
-  cd checkstyle-openrewrite-recipes
-  mvn -e --no-transfer-progress clean install -DskipTests
-
-  cd "$PROJECT_ROOT"
+# disable until: https://github.com/checkstyle/checkstyle-openrewrite-recipes/issues/116
```

Comment:
sorry, we need it, it is primary above all other recipies.
I have technical issues with ability to release, but I hope to do this in next 1-2 weeks

Reply:
then its no problem, thx.